tests/ctest/test_cmerge.c

```diff
@@ -302,6 +302,173 @@ static void testArrayWithDifferentLengths (void)
 	}
 }
 
+/**
+  * KeySets equality depends only on what is below root. Consequently the three sets
+  * user/test1 and user/test2 and user/test3 must be equal
```

Comment:
@markus2330 Is this ok?

Reply:
GitHub only shows me the comment and can you ask me a more concrete question so that I know what should be ok?